pkg/meta/meta.go

```diff
@@ -1560,7 +1578,71 @@ func (m *Meta) GetHistoryDDLJobsIterator(startJobID int64) (LastJobIterator, err
 
 // HLastJobIterator is the iterator for gets the latest history.
 type HLastJobIterator struct {
-	iter *structure.ReverseHashIterator
+	iter        *structure.ReverseHashIterator
+	schemaNames set.StringSet
+	tableNames  set.StringSet
+}
+
+func findNextNonEscapedQuote(s string) int {
+	findEscape := false
+	for i, ch := range s {
+		if ch == '\\' {
```

Comment:
for two consecutive `\`, the second one should not trigger `findEscape` (they are one escaped `\` in JSON string). Please add this test case.